src/components/AddCar.js

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable */
+
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { addCar } from '../redux/actions/CarAction';
+
+const AddCar = () => {
```

Comment:
- [ ] In the add car form, there is no validation which resulted in these:

![image](https://user-images.githubusercontent.com/57421823/224172742-5d6c8513-24f7-45e2-9b0f-387d957810d2.png)

I couldn't delete them either.

Kindly add some validation to that form, so this issue won't happen to your users. 👍